src/main/java/uk/gov/ons/census/casesvc/service/NewAddressReportedService.java

```diff
@@ -114,4 +137,89 @@ private void checkManadatoryFieldsPresent(CollectionCase newCollectionCase) {
+    newCase.setAddressType(newCollectionCase.getAddress().getAddressType());
+
+    // Set fields that come from source case, unless they are in the event
+    newCase.setAddressLine1(
+        getEventValOverSource(
+            sourceCase.getAddressLine1(), newCollectionCase.getAddress().getAddressLine1()));
+    newCase.setAddressLine2(
+        getEventValOverSource(
+            sourceCase.getAddressLine2(), newCollectionCase.getAddress().getAddressLine2()));
+    newCase.setAddressLine3(
+        getEventValOverSource(
+            sourceCase.getAddressLine3(), newCollectionCase.getAddress().getAddressLine3()));
+    newCase.setTownName(
+        getEventValOverSource(
+            sourceCase.getTownName(), newCollectionCase.getAddress().getTownName()));
+    newCase.setPostcode(
+        getEventValOverSource(
+            sourceCase.getPostcode(), newCollectionCase.getAddress().getPostcode()));
+    newCase.setCollectionExerciseId(
+        getEventValOverSource(
+            sourceCase.getCollectionExerciseId(), newCollectionCase.getCollectionExerciseId()));
+    newCase.setEstabType(
+        getEventValOverSource(
+            sourceCase.getEstabType(), newCollectionCase.getAddress().getEstabType()));
+    newCase.setFieldCoordinatorId(
+        getEventValOverSource(
+            sourceCase.getFieldCoordinatorId(), newCollectionCase.getFieldCoordinatorId()));
+    newCase.setFieldOfficerId(
+        getEventValOverSource(
+            sourceCase.getFieldOfficerId(), newCollectionCase.getFieldOfficerId()));
+
+    // Set fields empty/null/blank unless they come from the event
+    newCase.setOrganisationName(
+        getEventValOverSource(null, newCollectionCase.getAddress().getOrganisationName()));
+    newCase.setLatitude(getEventValOverSource(null, newCollectionCase.getAddress().getLatitude()));
+    newCase.setLongitude(
+        getEventValOverSource(null, newCollectionCase.getAddress().getLongitude()));
+    newCase.setUprn(getEventValOverSource(null, newCollectionCase.getAddress().getUprn()));
+    newCase.setCeExpectedCapacity(
+        getEventValOverSource(null, newCollectionCase.getCeExpectedCapacity()));
+    newCase.setCaseType(getEventValOverSource(null, newCollectionCase.getCaseType()));
+    newCase.setTreatmentCode(getEventValOverSource(null, newCollectionCase.getTreatmentCode()));
+
+    // Fields that do not come on the event but come from source case
+    newCase.setEstabUprn(sourceCase.getEstabUprn());
+    newCase.setAbpCode(sourceCase.getAbpCode());
+    newCase.setHtcDigital(sourceCase.getHtcDigital());
+    newCase.setHtcWillingness(sourceCase.getHtcWillingness());
+    newCase.setLad(sourceCase.getLad());
+    newCase.setLsoa(sourceCase.getLsoa());
+    newCase.setMsoa(sourceCase.getMsoa());
+    newCase.setOa(sourceCase.getOa());
+    newCase.setPrintBatch(sourceCase.getPrintBatch());
+    newCase.setSecretSequenceNumber(sourceCase.getSecretSequenceNumber());
```

Comment:
Don't copy this over - it's very important that Hibernate sets this because it's what guarantees that our case ref is unique